iib/web/static/api_v1.yaml

```diff
@@ -0,0 +1,461 @@
+                    type: string
+                    example: '"bundles" should be a non-empty array of strings'
+        '401':
+          description: The user is not allowed to create a request with this input
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: You must be authenticated to perform this action
+  /builds/rm:
+    post:
+      description: Submit a build request to remove operators from an index image
+      requestBody:
+        description: The build request to remove
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestRemove'
+      security:
+        - Kerberos Authentication: []
+      responses:
+        '201':
+          description: The build request was initiated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RequestVerbose'
+        '400':
+          description: The input is invalid
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: '"operators" should be a non-empty array of strings'
+        '401':
+          description: The user is not allowed to submit a build request with this input
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: You must be authenticated to perform this action
+components:
+  schemas:
+    BundleMapping:
+      type: object
+      example:
+        some_operator:
+          - registry.com/op/go-logging
+      properties:
+        operator:
+          type: array
+          items:
+            type: string
+          example:
+            - some_bundle
+            - some_other_bundle
+    Pagination:
+      type: object
+      properties:
+        first:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=1&per_page=20&verbose=False
+        last:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=3&per_page=20&verbose=False
+        next:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=1&per_page=20&verbose=False
+        page:
+          type: integer
+          example: 1
+        pages:
+          type: integer
+          example: 3
+        per_page:
+          type: integer
+          example: 20
+        previous:
+          type: string
+          example: null
+        total:
+          type: integer
+          example: 45
+    Request:
```

Comment:
`index_image` is missing.

Reply:
Isn't it `from_index` and not `index_image`?
`from_index` is in there :)

Reply:
No, I mean `index_image.  :smile: